spec/integration/generators/blacklight/solr5_generator_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+require 'generators/blacklight/solr5_generator'
+
+describe Blacklight::Solr5Generator do
```

Comment:
I thought it was worth it since the generator methods depend on the solr directory and .solr_wrapper.yml being in `Blacklight.root`, not in the standard generators template directory. I could see future changes moving the solr directory and breaking this generator. 

But if it feels like overkill, I can remove the test.